.evergreen/generate_evergreen_tasks.js

```diff
@@ -388,12 +403,21 @@ for (const {
     return !isAWSTask && !isSkippedTaskOnWindows;
   });
 
+  function npmVersion(nodeVersion) {
+    switch (nodeVersion) {
+      case 16:
+      case 14: return 9;
+      case 12: return 8;
```

Comment:
npm@9 does not work with Node 12

Reply:
https://spruce.mongodb.com/task/mongo_node_driver_rhel80_large_erbium_test_3.6_replica_set_patch_cdc3d53f26d0701a22337025e2e96fda070badb6_6501b8bbe3c33107e3236598_23_09_13_13_27_24/logs?execution=0

somehow 12 still picked up npm 9?

Reply:
Ah yes the default node version is 12 so when NODE_LTS_VERSION is not defined I've defaulted npm to be 8 now.